compose/ui/ui-test-junit4/src/jvmMain/kotlin/androidx/compose/ui/test/junit4/ComposeTestRule.jvm.kt

```diff
@@ -251,6 +252,7 @@ interface ComposeContentTestRule : ComposeTestRule {
      *
      * @throws IllegalStateException if called more than once per test.
      */
+    @UiComposable
     fun setContent(composable: @Composable () -> Unit)
```

Comment:
I think you want this instead. This function can be called from anywhere, it's only the lambda itself which needs to be Compose UI.
```suggestion
    fun setContent(composable: @Composable @UiComposable () -> Unit)
```

Reply:
Ahh good catch 🤦‍♂️